export/usb.py

```diff
@@ -3,13 +3,18 @@
 import re
 from subprocess import CalledProcessError, check_call
 import logging
-
 from pyudev import Context, Monitor
```

Comment:
pyudev is a 3rd party library. There should be a space between it and Python Standard Library imports